tests/e2e/util.go

```diff
@@ -1611,6 +1612,66 @@ func getWCPSessionId(hostname string, username string, password string) string {
 
 }
 
+//getk8sWindowsWorkerIPs returns the windows worker node IPs
+func getk8sWindowsWorkerIPs(ctx context.Context, client clientset.Interface, nodeName string) string {
+	var err error
+	nodes, err := client.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	var windowsWorkerIp string
+	for _, node := range nodes.Items {
+		if node.Name == nodeName {
+			addrs := node.Status.Addresses
+			for _, addr := range addrs {
+				if addr.Type == v1.NodeInternalIP && (net.ParseIP(addr.Address)).To4() != nil {
+					windowsWorkerIp = addr.Address
+				}
+			}
+		}
+	}
+        gomega.Expect(windowsWorkerIp).NotTo(gomega.BeEmpty(), "Unable to find k8s windows worker IP")
+        return windowsWorkerIp
+	
+} 
+
+//execCommanOnWindowsWorker func returns the size of the pod
+func execCommanOnWindowsWorker(ctx context.Context, client clientset.Interface, windowsWorkerIP string) int64{
+	sshClient, err := simplessh.ConnectWithPassword(windowsWorkerIP,"Administrator",esxPassword)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	defer sshClient.Close()
+	
+	cmd := "Get-Disk | Format-List -Property Manufacturer,Size"
+	framework.Logf("command to be executed in windows node %s", cmd)
+	output, err := sshClient.Exec(cmd)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	framework.Logf("GetDisk output %s\n",string(output))
```

Comment:
may be move it before we check for error, so that we know what was the error from the powershell command
also, change the log message a little to "GetDisk output:\n%s"

Reply:
done